eng/Versions.props

```
@@ -219,7 +219,7 @@
     <!-- ICU -->
     <MicrosoftNETCoreRuntimeICUTransportVersion>8.0.0-rtm.23523.2</MicrosoftNETCoreRuntimeICUTransportVersion>
     <!-- MsQuic -->
-    <MicrosoftNativeQuicMsQuicVersion>2.2.3</MicrosoftNativeQuicMsQuicVersion>
+    <MicrosoftNativeQuicMsQuicVersion>2.2.5-ci.444313</MicrosoftNativeQuicMsQuicVersion>
```

@dotnet/ncl this package version came from internal but it does not exist in public. I see this version was pushed 20 mins ago: https://www.nuget.org/packages/Microsoft.Native.Quic.MsQuic.Schannel/2.3.5 is that the one we need to use?

---

```suggestion
    <MicrosoftNativeQuicMsQuicVersion>2.3.5</MicrosoftNativeQuicMsQuicVersion>
```

---

Ah, no, that one is "Schannel". It's a different one.

---

Actually, that one is the correct one. The name is misleading, but it's the package we need, according to https://github.com/dotnet/runtime/blob/release/8.0/src/libraries/System.Net.Quic/src/System.Net.Quic.csproj#L150

```suggestion
    <MicrosoftNativeQuicMsQuicVersion>2.3.5</MicrosoftNativeQuicMsQuicVersion>
```